azure/resource_groups/app/parameters/production.template.json

```
@@ -142,10 +142,10 @@
       }
     },
     "MAINTENANCE_MODE": {
-      "value": ""
+      "value": "1"
     },
     "MAINTENANCE_MODE_AVAILABILITY_MESSAGE": {
-      "value": ""
+      "value": "Private beta has ended. The service will be back in November."
```

Updated content from the team:

```suggestion
      "value": "It's not possible to make a claim at the moment. The service will be back in November."
```